test/configparser.cpp

```diff
@@ -248,6 +248,29 @@ TEST_CASE("\"unbind-key -a\" removes all key bindings", "[ConfigParser]")
 	}
 }
 
+TEST_CASE("Include config file with multi line macros", "[ConfigParser]")
+{
+	ConfigParser cfgparser;
+	SECTION("Successfully includes & parse file with multi-line macros") {
```

Comment:
There's no need to create a section when we're only testing a single scenario; please remove it.